site/app/config/config.yml

```diff
@@ -199,3 +199,19 @@ sentry:
     dsn: "%sentry_dsn%"
     options:
         environment: "%sentry_env%"
+
+# algolia search indices configuration
+algolia_search:
+  prefix: ''
+  indices:
+    - name: projects
+      class: Librecores\ProjectRepoBundle\Entity\Project
+
+    - name: classifications
+      class: Librecores\ProjectRepoBundle\Entity\ProjectClassification
```

Comment:
Is it really necessary to specify the classifications as separate index, doesn't this go into the projects index?